components/api-resource-mgt/org.wso2.carbon.identity.api.resource.collection.mgt/src/main/java/org/wso2/carbon/identity/api/resource/collection/mgt/model/APIResourceCollectionSearchResult.java

```diff
@@ -0,0 +1,32 @@
+package org.wso2.carbon.identity.api.resource.collection.mgt.model;
+
+import java.util.List;
+
+/**
+ * API Resource Collection Search Result.
+ */
+public class APIResourceCollectionSearchResult {
```

Comment:
Let's add a new line

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/5807.